spk/flexget/Makefile

```diff
@@ -1,16 +1,19 @@
 SPK_NAME = flexget
-SPK_VERS = 2.9.14
-SPK_REV = 6
+SPK_VERS = 3.1.77
+SPK_REV = 7
 SPK_ICON = src/flexget.png
 
+REQUIRED_DSM = 6.0
```

Comment:
I would not use REQUIRED_DSM alas there is a good reason for this. (if the reason is support for python3>=3.7.7, then this requirement is already defined by SPK_DEPENDS)